app/src/layouts/calendar/index.ts

```diff
@@ -41,8 +41,12 @@ export default defineLayout<LayoutOptions>({
 		const appStore = useAppStore();
 
 		const layoutOptions = useSync(props, 'layoutOptions', emit);
-		const filter = useSync(props, 'filter', emit);
-		const search = useSync(props, 'search', emit);
+		const filter = useSync(props, 'filter', () => {
+			// no emit
+		});
+		const search = useSync(props, 'search', () => {
+			// no emit
+		});
```

Comment:
Not sure about this... Filter and search still works in the app, but there might be a better solution.

Thrown error was:
```
Error: app/src/layouts/calendar/index.ts(44,43): error TS2345: Argument of type '(event: "update:selection" | "update:layoutOptions" | "update:layoutQuery", ...args: any[]) => void' is not assignable to parameter of type '(event: "update:filter", ...args: any[]) => void'.
  Types of parameters 'event' and 'event' are incompatible.
    Type '"update:filter"' is not assignable to type '"update:selection" | "update:layoutOptions" | "update:layoutQuery"'.
Error: app/src/layouts/calendar/index.ts(45,43): error TS2345: Argument of type '(event: "update:selection" | "update:layoutOptions" | "update:layoutQuery", ...args: any[]) => void' is not assignable to parameter of type '(event: "update:search", ...args: any[]) => void'.
  Types of parameters 'event' and 'event' are incompatible.
    Type '"update:search"' is not assignable to type '"update:selection" | "update:layoutOptions" | "update:layoutQuery"'.
```

Reply:
These are no long a syncable thing. They've been moved to the "toRefs" bit below 👍🏻